content/en/docs/concepts/architecture/nodes.md

```diff
@@ -28,7 +33,7 @@ The [components](/docs/concepts/overview/components/#node-components) on a node
 
 ## Management
 
-There are two main ways to have Nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
+There are two main ways to have Node objects added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
```

Comment:
This is correct, but not necessary.

Reply:
But that too creates confusion, Isn't it?

Reply:
In this context, it is obvious you are not supposed to add a new `Node` kind to the API server.
Users know what you are talking about. Right?

Reply:
@sftim what's your thoughts about it?